tests/test_auth_helper.py

```diff
@@ -38,23 +40,144 @@ def mock_builtin_open(unstub):
     unstub()
 
 
+def test_get_or_refresh_access_token_both_valid(mock_cli_config):
```

Comment:
is there a test for the scenario where `_load_local_token` fails specifically because the access token is expired?  I dont think there is one in this pr but there is probably already an existing one

Reply:
yes, it'd be through _validate_local_token returning None when the token is expired, and we test that in the [`test_validate_token_expired` test in this file](https://github.com/DataBiosphere/terra-scientific-pipelines-service-cli/pull/11/files#diff-c866071a7d19251141cab924e1b64f7b25c15467689f71ae97bb9e753d7b742fL148)